### PR TITLE
Moved all scripts in all html files to head with defer attribute

### PR DESCRIPTION
**Pull request by briosolivares**
#60 
I believe this is the formatting we wanted, having all the assets in one place in the HTML files. What do you think?

### CODE REVIEW

**Comment by j0n3lson** [comment]
Hi  @Bcoder19, 

 A random law from [Hacker Laws repository](https://github.com/dwmkerr/hacker-laws) is a small gift to thank you for opening the PR.

-----------------------------------------------------
The Pareto Principle (The 80/20 Rule)
-----------------------------------------------------

The Pareto Principle on Wikipedia

> Most things in life are not distributed evenly.

The Pareto Principle suggests that in some cases, the majority of results come from a minority of inputs:

- 80% of a certain piece of software can be written in 20% of the total allocated time (conversely, the hardest 20% of the code takes 80% of the time)
- 20% of the effort produces 80% of the result
- 20% of the work creates 80% of the revenue
- 20% of the bugs cause 80% of the crashes
- 20% of the features cause 80% of the usage

In the 1940s American-Romanian engineer Dr. Joseph Juran, who is widely credited with being the father of quality control, began to apply the Pareto principle to quality issues.

This principle is also known as: The 80/20 Rule, The Law of the Vital Few, and The Principle of Factor Sparsity.

Real-world examples:

- In 2002 Microsoft reported that by fixing the top 20% of the most-reported bugs, 80% of the related errors and crashes in windows and office would become eliminated (Reference).

-----------------------------------------------------
         github.com/dwmkerr/hacker-laws by Dave Kerr 
-----------------------------------------------------

File: public/templates/finesse_match.html
Comment (Steve-Leke) [inline public/templates/finesse_match.html:48]: Just a heads up. This file will be deleted once my pr gets merged into the main repo. This file name gets changed to "demo.html". I'd say wait till my pr gets merged and then do a fetch/merge/rebase, and add these changes to that file.

File: public/templates/firebase.html
Comment (Steve-Leke) [inline public/templates/firebase.html:66]: Hey,

I created an issue asking for the jquery to be updated (#68). You don't have to do the plugins but editing all the jquery stuff could be done in this pr. What do you think @j0n3lson?

Comment (j0n3lson) [reply public/templates/firebase.html:66]: No, let's do the jQuery update in a separate PR so we have a small scope change for this PR (e.g. it's only about moving the script statements).

Let's leave this as is for now.

File: public/templates/finesse_match.html
Comment (j0n3lson) [reply public/templates/finesse_match.html:48]: Let's move forward with this change. Steve, you'll have to resolve the merge conflict on your PR.

**Review by j0n3lson — APPROVED** [review]
This looks good to me! Going to merge it in! Great job.

**Pull request merged**
